libc/newhdrgen/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+project(newhdrgen LANGUAGES CXX)
+
+enable_testing()
+
+set(NEWHDGEN_TESTS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/tests)
+
+add_test(
```

Comment:
add only when in full build mode

Not sure if this is correct but: if(CMAKE_BUILD_TYPE STREQUAL FULL) /..../ endif()

Reply:
Will fix!